src/client/Microsoft.Identity.Client/ManagedIdentity/ManagedIdentityRequest.cs

```diff
@@ -4,18 +4,13 @@
 using System;
 using System.Collections.Generic;
 using System.Net.Http;
-using Microsoft.Identity.Client.Http;
 using Microsoft.Identity.Client.Utils;
+using static Microsoft.Identity.Client.Http.Retry.IRetryPolicyFactory;
 
 namespace Microsoft.Identity.Client.ManagedIdentity
 {
     internal class ManagedIdentityRequest
     {
-        // referenced in unit tests, cannot be private
-        public const int DefaultManagedIdentityMaxRetries = 3;
-        // this will be overridden in the unit tests so that they run faster
-        public static int DefaultManagedIdentityRetryDelayMs { get; set; } = 1000;
```

Comment:
you need to rebase

Reply:
Do you mean merge the latest from the main branch? If so, done.